libkbfs/folder_branch_ops.go

```diff
@@ -81,7 +81,7 @@ const (
 	maxMDsAtATime = 10
 	// Time between checks for dirty files to flush, in case Sync is
 	// never called.
-	secondsBetweenBackgroundFlushes = 10
+	secondsBetweenBackgroundFlushes = 1
```

Comment:
I wonder if it makes sense to pause the ticker for background flusher when no outstanding data is left to flush, and re-enable it when new requests come in? Otherwise this would increase the idle resource usage